messagerouter/commands.go

```diff
@@ -75,14 +89,19 @@ func hasPrefix(s string) bool {
 // Handle registers the handler for the given command
 // Must match a command prefix, `!` for admin commands, `%` for general commands & `#` for MUD commands
 func (mr *MessageRouter) Handle(cmdString string, fn MessageHandler) error {
-	if !hasPrefix(cmdString) {
+	if !hasPrefix(cmdString) && cmdString != "*" {
```

Comment:
It's not really a command prefix, it's to tell the Handle function that it's a function to parse all messages. Perhaps needs its own handle function for clarity?